tests/conftest.py

```diff
@@ -1492,21 +1491,11 @@ def finalizer():
                     or mcg_only_deployment
                     or not ceph_cluster_installed
                 ):
-                    ceph_health_retry = False
-                    for mark in node.iter_markers():
-                        if "ceph_health_retry" == mark.name:
-                            ceph_health_retry = True
-                    if ceph_health_retry:
-                        ceph_health_check(
-                            namespace=config.ENV_DATA["cluster_namespace"]
-                        )
-                        log.info(
-                            "Ceph health check passed at teardown. (After test "
-                            "marked with @ceph_health_retry. For such TC we allow more re-tries)"
-                        )
-                    else:
-                        ceph_health_check_base()
-                        log.info("Ceph health check passed at teardown")
+                    ceph_health_check(namespace=config.ENV_DATA["cluster_namespace"])
```

Comment:
this will return True as soon as health is OK. but the issue we have is flip-flopping of ceph health. After upgrade health is OK and by the time it comes to teardown health is in HEALTH_WARN. either we can handle that in the test cases itself like ceph health should be stable after upgrade for some consecutive retries or in teardown.

Reply:
the discussion is the same as for master PR: https://github.com/red-hat-storage/ocs-ci/pull/8918#discussion_r1403077647